{
  "schema_version": "1.2.0",
  "id": "GHSA-hx5h-h8m3-hvw4",
  "modified": "2022-07-12T00:00:55Z",
  "published": "2022-07-02T00:00:20Z",
  "aliases": [
    "CVE-2022-32082"
  ],
  "details": "MariaDB v10.5 to v10.7 was discovered to contain an assertion failure at table->get_ref_count() == 0 in dict0dict.cc.",
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
    }
  ],
  "affected": [

  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2022-32082"
    },
    {
      "type": "WEB",
      "url": "https://jira.mariadb.org/browse/MDEV-26433"
    }
  ],
  "database_specific": {
    "cwe_ids": [
      "CWE-617"
    ],
    "severity": "HIGH",
    "github_reviewed": false
  }
}